Dockerfile

```
@@ -29,11 +21,9 @@ COPY ${APP_DIR}/internal ./internal
 USER 0
 RUN make build
 
-#@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-ose-cli:v4.9)
-FROM quay.io/openshift/origin-cli:4.12 AS origincli
+FROM quay.io/openshift/origin-cli:4.13 AS origincli
 
-#@follow_tag(registry.redhat.io/ubi8:latest)
-FROM registry.redhat.io/ubi8:8.7
+FROM registry.redhat.io/ubi9/ubi:9.1
```

We're already using 9.2 in our midstream builds. My suggestion would be to use "latest" as the tag to get the newest version:

```suggestion
FROM registry.redhat.io/ubi9/ubi:latest
```